source/actions3/actions/CLSSIS3820ScalerDarkCurrentMeasurementActionInfo.h

```diff
@@ -0,0 +1,37 @@
+#ifndef CLSSIS3820SCALERDARKCURRENTMEASUREMENTACTIONINFO_H
+#define CLSSIS3820SCALERDARKCURRENTMEASUREMENTACTIONINFO_H
+
+#include "actions3/AMActionInfo3.h"
+#include "beamline/CLS/CLSSIS3820Scaler.h"
+
+class CLSSIS3820ScalerDarkCurrentMeasurementActionInfo : public AMListActionInfo3
+{
+    Q_OBJECT
+
+public:
+	/// Constructor.
+	Q_INVOKABLE CLSSIS3820ScalerDarkCurrentMeasurementActionInfo(CLSSIS3820Scaler *scaler, double secondsDwell, QObject *parent = 0);
```

Comment:
This is more of a question for @davidChevrier.  Although this is definitely the most convenient and fast way to get a hold of the scaler, won't this cause issues with database loading?  This would need access to the beamline for loading which is generally something we try to avoid with all info and dataman classes.  I think we may need to add a generic way to handle getting the scaler from beamline classes.